cmd/query/app/http_handler.go

```diff
@@ -157,13 +157,28 @@ func (aH *APIHandler) getOperationsLegacy(w http.ResponseWriter, r *http.Request
 	vars := mux.Vars(r)
 	// given how getOperationsLegacy is bound to URL route, serviceParam cannot be empty
 	service, _ := url.QueryUnescape(vars[serviceParam])
-	operations, err := aH.queryService.GetOperations(r.Context(), service)
+	//for backwards compatibility, we will retrieve operations with all span kind
```

Comment:
* the accepted practice (even enforced by some linters) is to leave a space after `//`
* all span kinds